src/core/sys/dragonflybsd/dlfcn.d

```diff
@@ -14,8 +14,6 @@ version (DragonFlyBSD):
 
 extern (C) nothrow @nogc @system:
 
-enum __BSD_VISIBLE = true;
-
```

Comment:
Note, this might look like a massive edit, but:
- it only removed checking for __BSD_VISIBLE / __POSIX_VISIBLE (which are defined in the cdefs.d file) in these subdirectories
- re-indent the lines